eif_utils/src/lib.rs

```diff
@@ -711,8 +712,8 @@ impl<T: Digest + Debug + Write + Clone> EifBuilder<T> {
         meta.insert("BuildTool".to_string(), tool_name);
         meta.insert("BuildToolVersion".to_string(), tool_version);
 
-        let config_file =
-            File::open(KERNEL_CONFIG_PATH).expect("Failed to open kernel image config file.");
+        let config_file = File::open(format!("{}/bzImage.config", blobs_path))
```

Comment:
the name of the config differs by platform

Reply:
I changed the names to match the platform and also changed the blob names in samples/command_executer/resources/blobs